etc/jitsi-meet-desktop.profile

```diff
@@ -0,0 +1,30 @@
+# Firejail profile for jitsi-meet-desktop
+# Description: Jitsi Meet desktop application powered by Electron
+# This file is overwritten after every install/update
+# Persistent local customizations
+include jitsi-meet-desktop.local
+# Persistent global definitions
+include jitsi-meet-desktop.local
+
+noblacklist ${HOME}/.config/Jitsi Meet
+
+include disable-common.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+whitelist ${DOWNLOADS}
+mkdir ${HOME}/.config/Jitsi Meet
+whitelist ${HOME}/.config/Jitsi Meet
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+
+private-tmp
```

Comment:
What about 
- `disable-mnt`
- `private-cache`
- `private-bin`
- `private-etc`
- `private-dev` (if it not break cam/micro)

Reply:
Jitsi Meet doesn't seem to use the ```.cache``` directory, so I have not used ```private-cache```. There is a cache directory in ```.config/Jitsi Meet```, which is what the application seems to use.

Reply:
For ```private-etc```, I still haven't been able to figure out which files in ```/etc/``` it expects to be able to access. I have only been able to get it to show a blank window till now. I'll add it the option when I figure that out.

Reply:
You can use `firejail --build path/to/jitsi-meet-desktop`.

> Jitsi Meet doesn't seem to use the .cache directory, so I have not used private-cache.

`private-cache` means also hide the the caches of other apps. (This is also done by whitelist, but if a user deactivates whitelist the cache is still protected)

Reply:
The manpage says under ```---build``` that Chromium-based browsers won't work, which I guess explains why the program doesn't launch with ```firejail --build```.

Any ideas on how one would figure out which filesin ```/etc``` it would need?

Reply:
`--trace` will also break, you could try it with the templates.

Try this (still need ordering), of it does not work leave it out.
`private-etc alternatives,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,mime.types,xdg,passwd,group,ca-certificates,ssl,pki,crypto-policies,nsswitch.conf,resolv.conf,hosts,host.conf,hostname,protocols,services,rpc,alsa,asound.conf,pulse,machine-id,fonts,pango,X11,drirc,glvnd,bumblebee,nvidia`

Reply:
The one you suggested works. Thanks a lot.